pkg/codegen/templates/typedef.tmpl

```diff
@@ -56,7 +56,7 @@ func (i *ServerImpl) {{printf "gnmiGet%s" .TypeName}}(ctx context.Context,
 		return nil, fmt.Errorf("unexpected type of reply from server %v", gnmiVal.Value)
 	}
 
-	log.Infof("gNMI Json %s", string(gnmiJsonVal.JsonVal))
+	log.Debugf("gNMI Json %s", string(gnmiJsonVal.JsonVal))
```

Comment:
do you want this back to Info

Reply:
I thought it was too verbose - it prints out the whole message - what do you think?

Reply:
So long as it's easy to override the verbosity level (for example, from a helm chart /. values override), I think it's fine.

Reply:
👍

Reply:
You can change it in the `helm-charts` since the last couple weeks: https://gerrit.opencord.org/c/roc-helm-charts/+/31404/2/aether-roc-api/templates/deployment.yaml

Reply:
Sweet :+1: